integrationTests/vm/txsFee/guardAccount_test.go

```diff
@@ -117,9 +153,55 @@ func setGuardian(testContext *vm.VMTestContext, userAddress []byte, guardianAddr
 		gasLimit,
 		[]byte("SetGuardian@"+hex.EncodeToString(guardianAddress)+"@"+hex.EncodeToString(uuid)))
 
-	if isInstantSet {
-		tx.GuardianAddr = guardianAddress
-	}
+	return testContext.TxProcessor.ProcessTransaction(tx)
+}
+
+func setGuardianCoSigned(
+	testContext *vm.VMTestContext,
+	userAddress []byte,
+	currentGuardianAddress []byte,
+	newGuardianAddress []byte,
+	uuid []byte,
+) (vmcommon.ReturnCode, error) {
+	gasPrice := uint64(10)
+	gasLimit := uint64(250000 + 50000 + 1000)
```

Comment:
maybe you can add a const for `gasLimit` ?

Reply:
added consts